src/settings/ClosingReasons.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import ClosingReasonsForm from './ClosingReasonsForm';
+
+
+class ClosingReasons extends Component {
+  static manifest = Object.freeze({
+    recordId: {},
+    closingReasons: {
+      type: 'okapi',
+      records: 'configs',
+      path: 'configurations/entries',
+      GET: {
+        params: {
+          query: '(module=ORDERS and configName=closing-reasons)',
+        },
+      },
+      POST: {
+        path: 'configurations/entries',
+      },
+      PUT: {
+        path: 'configurations/entries/%{recordId}',
+      },
+      DELETE: {
+        path: 'configurations/entries/%{recordId}',
+      }
+    },
+  });
+
+  static propTypes = {
+    label: PropTypes.string.isRequired,
+    resources: PropTypes.shape({
+      closingReasons: PropTypes.object.isRequired,
+    }),
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.styles = {
+      closingReasonsWrapper: {
+        width: '100%',
+      },
+    };
+  }
+
+  render() {
+    const { label, resources } = this.props;
+
+    return (
+      <div style={this.styles.closingReasonsWrapper}>
+        <ClosingReasonsForm
+          initialValues={resources.closingReasons}
```

Comment:
I think you should extract this to variable, like `const initialValues = get(resources, 'closingReasons', []);`